src/bots/filler.ts

```diff
@@ -897,6 +900,72 @@ export class FillerBot implements Bot {
 		return true;
 	}
 
+	protected selectSubsetOfMakers(nodeToFill: NodeToFill) {
+		let makerNodes = nodeToFill.makerNodes;
```

Comment:
does the sdk's `findPerpNodesToFill` et al. need to be updated to include more makers? i think the dlob code only returns enough makers to completely fill the taker